packages/graphback-db-manage/tests/snapshots/GraphQLSchemaManager.test.ts.md

```diff
@@ -0,0 +1,44 @@
+# Snapshot report for `tests/GraphQLSchemaManager.test.ts`
+
+The actual snapshot is saved in `GraphQLSchemaManager.test.ts.snap`.
+
+Generated by [AVA](https://ava.li).
+
+## it should detect multiple changes
+
+> Snapshot 1
+
+    `[␊
+      {␊
+        "criticality": {␊
+          "level": "NON_BREAKING"␊
+        },␊
+        "type": "TYPE_ADDED",␊
+        "message": "Type 'Test' was added",␊
+        "path": "Test"␊
+      },␊
+      {␊
+        "criticality": {␊
+          "level": "NON_BREAKING"␊
+        },␊
+        "type": "FIELD_ADDED",␊
+        "message": "Field 'age' was added to object type 'User'",␊
+        "path": "User.age"␊
+      }␊
+    ]`
+
+## it should update the old schema to match the new one
```

Comment:
Not sure I get this case? This is about database so I would like to see how input (2 schemas) affects output (SQL files). Case above is good but to low level